spanner/spring-data/src/main/com/example/spanner/SpannerSchemaToolsSample.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.spanner;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.cloud.gcp.data.spanner.core.admin.SpannerDatabaseAdminTemplate;
+import org.springframework.cloud.gcp.data.spanner.core.admin.SpannerSchemaUtils;
+
+/**
+ * This sample demonstrates how to generate schemas for interleaved tables from POJOs and how to
+ * execute DDL.
+ */
+//[START spring_data_spanner_schema_sample]
+@SpringBootApplication
+public class SpannerSchemaToolsSample {
```

Comment:
See my comment on `SpannerRepositorySample`.

Reply:
as discussed separately, using the component annotation so that the 3 separate files for snippets are separated.